packages/integrations/svelte/src/index.ts

```diff
@@ -1,13 +1,16 @@
 import type { Options } from '@sveltejs/vite-plugin-svelte';
 import { svelte, vitePreprocess } from '@sveltejs/vite-plugin-svelte';
+import { VERSION } from 'svelte/compiler';
 import type { AstroIntegration, AstroRenderer } from 'astro';
 import { fileURLToPath } from 'node:url';
 import type { UserConfig } from 'vite';
 
+const isSvelte5 = VERSION.startsWith('5');
```

Comment:
Paranoid future-proofing.

```suggestion
const isSvelte5 = Number.parseInt(VERSION.split('.').at(0)) >= 5;
```